contracts/SuiBridge.sol

```diff
@@ -3,7 +3,6 @@ pragma solidity ^0.8.20;
 
 import "@openzeppelin/contracts-upgradeable/utils/PausableUpgradeable.sol";
 import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
we still need this correct?

Reply:
You are right, we need/use it, but it is inherited from: `import "./interfaces/IWETH9.sol";`